### PR TITLE
docs: Support setting global node runtime

### PR DESCRIPTION
**Pull request by tianfeng92**
### Description
Add docs for configurable node.js version.

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/web-apps/automated-testing/_partials/_advanced-nodejs.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Using Node.js Runtime on Sauce Cloud
+
+Since `saucectl` v0.185.0 and above, you can specify Node.js runtime version on Sauce Cloud.
+
+Only supported test runners can use configurable Node.js versions. Check the following pages for supported framework versions that allow configurable Node.js versions.
```
Comment (mhan83) [inline docs/web-apps/automated-testing/_partials/_advanced-nodejs.md:5]:
```suggestion
Only supported test runners can use configurable Node.js versions. Check the following pages for supported framework versions.
```

Nit for brevity.

File: docs/web-apps/automated-testing/_partials/_advanced-nodejs.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- [TestCafe](../testcafe.md#supported-testing-platforms)
+
+### Supported Node.js Versions
+
+Sauce Labs supports the following Node.js versions:
+
+<table id="table-nodejs">
+  <tr>
+    <th>Node.js Version</th>
+    <th>Alias Name</th>
```
Comment (mhan83) [inline docs/web-apps/automated-testing/_partials/_advanced-nodejs.md:19]:
```suggestion
    <th>Alias</th>
```

File: docs/web-apps/automated-testing/cypress.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Sauce Labs supports the following test configurations for Cypress:
   <tbody>
     <tr>
       <td rowspan='2'>13.13.3</td>
-      <td rowspan='2'>20</td>
+      <td rowspan='2'>20, Version Configurable</td>
```
Comment (mhan83) [inline docs/web-apps/automated-testing/cypress.md:40]: What do you think of adding a new column for this? 

| Cypress Version | Node.js Version | Supports configurable runtime |
| -----------------| --------------- | ------------------------------- |
| 13.13.3 | 20 | ✅ |

Comment (tianfeng92) [reply docs/web-apps/automated-testing/cypress.md:40]: 👍 I was a bit hesitant about adding an extra column. Looks like good to have one.

**Review by mhan83 — APPROVED** [review]

**Review by alexplischke — APPROVED** [review]

**Pull request merged**